### ISSUE TITLE
Crash on non-pgp file for decryption

### ISSUE THREAD

**Issue by dschuermann** [open-keychain/open-keychain#1390]
This bug happens in current master but not in our current stable release 3.3.
I think this has been introduced by the operations pull request.

Reproduce:
Select non-pgp file for decryption

```
  8770         AndroidRuntime  E  FATAL EXCEPTION: Thread-12055
  8770         AndroidRuntime  E  Process: org.sufficientlysecure.keychain.debug, PID: 8770
  8770         AndroidRuntime  E  java.lang.ArrayIndexOutOfBoundsException: length=128; index=227
  8770         AndroidRuntime  E  at org.spongycastle.bcpg.ArmoredInputStream.decode(ArmoredInputStream.java:86)
  8770         AndroidRuntime  E  at org.spongycastle.bcpg.ArmoredInputStream.read(ArmoredInputStream.java:451)
  8770         AndroidRuntime  E  at org.spongycastle.bcpg.BCPGInputStream.nextPacketTag(BCPGInputStream.java:114)
  8770         AndroidRuntime  E  at org.spongycastle.openpgp.PGPObjectFactory.nextObject(PGPObjectFactory.java:101)
  8770         AndroidRuntime  E  at org.sufficientlysecure.keychain.pgp.PgpDecryptVerify.decryptVerify(PgpDecryptVerify.java:318)
  8770         AndroidRuntime  E  at org.sufficientlysecure.keychain.pgp.PgpDecryptVerify.executeInternal(PgpDecryptVerify.java:153)
  8770         AndroidRuntime  E  at org.sufficientlysecure.keychain.pgp.PgpDecryptVerify.execute(PgpDecryptVerify.java:115)
  8770         AndroidRuntime  E  at org.sufficientlysecure.keychain.pgp.PgpDecryptVerify.execute(PgpDecryptVerify.java:77)
  8770         AndroidRuntime  E  at org.sufficientlysecure.keychain.service.KeychainNewService$1.run(KeychainNewService.java:111)
  8770         AndroidRuntime  E  at java.lang.Thread.run(Thread.java:818)
  741         ActivityManager  W  Force finishing activity 1 org.sufficientlysecure.keychain.debug/org.sufficientlysecure.keychain.ui.DecryptActivity
```

**Comment by Valodim** [comment]
this actually seems file dependent. I'll look into it

**Comment by dschuermann** [comment]
Looks like I was wrong. This also happens in the current release.

**Comment by Valodim** [comment]
it probably happens on specific files where the first few bytes randomly look like pgp and it crashes from the assumptions made on the files afterwards

**Comment by Valodim** [comment]
fixed f1a75a81e724566e8f4e9c0e264ca548bca03abb
